doc/source/whatsnew/v0.25.0.rst

```diff
@@ -97,18 +97,18 @@ would be reassigned as -1. (:issue:`19387`)
 
 .. code-block:: ipython
 
-    MultiIndex(levels=[[nan, None, NaT, 128, 2]],
-               codes=[[0, -1, 1, 2, 3, 4]])
-    MultiIndex(levels=[[1, 2]],
-               codes=[[0, -2]])
+    Out[1]: MultiIndex(levels=[[nan, None, NaT, 128, 2]],
```

Comment:
Not quite. I think you want to change the `..ipython:: python` block up above to assign the two MIs

```
mi1 = pd.MultiIndex(...)
mi2 = pd.MultiIndex(...)
```

Then this would be `.. code-block:: ipython` like

    In [1]: mi1
    Out[1]...

    In [2]: mi2
    Out[2]: ...

And then below, you'd leave it as a `..ipython:: python` and just have the input,

```
    mi1
    mi2
```

which will be run.